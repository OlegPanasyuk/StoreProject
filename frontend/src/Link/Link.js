import React, { Component } from 'react';

class Link extends Component {
    constructor(props) {
        super(props);
      }
    
    render() {
        let { text, href } = this.props;
        return (
            <a href={href}>
                {text}
            </a>
        );
    }
}

export default Link;