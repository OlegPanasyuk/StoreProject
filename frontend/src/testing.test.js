import React from 'react';
import Link from './Link/Link';
import renderer from 'react-test-renderer';
import enzyme, { configure } from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


test('Link must be' , () => {
    const element = enzyme.shallow(
        <Link href='#' text='Go Workw'></Link>,
    );
    
    let j = shallowToJson(element);
    expect(j).toMatchSnapshot();
});



// test('2+2=4', () => {
//     expect(2 + 2).toBe(4);
// })