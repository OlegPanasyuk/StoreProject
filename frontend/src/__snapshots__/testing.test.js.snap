// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link must be 1`] = `
<a
  href="#"
>
  Go Workw
</a>
`;
